malasakit-django/pcari/templates/base.html

```diff
@@ -36,6 +37,18 @@
         <h1 id="main-heading">{% block main-heading %}Malasakit{% endblock %}</h1>
       </div>
     </header>
+    <page-summary>
+      <div class="container">
+        <center>
+          <p id="page-summary">
+            {% block page-summary %}
+              {% blocktrans trimmed %}
+              {% endblocktrans %}
+            {% endblock %}
+          </p>
+        </center>
+      </div>
+    </page-summary>
```

Comment:
Please remove this block. From what I can tell, the `page-summary` is just an abstraction for a paragraph, which adds a lot more complexity (i.e. another block we need to remember to use) for the comparatively small amount of boilerplate you've removed (i.e. `<p></p>`, which is seven characters).

Also, not all page summaries are alike, so they can't all share an ID `#page-summary`: I would like the `#privacy-notice` to be bolded, for example.